crypto/cryptlib.c

```diff
@@ -8,6 +8,10 @@
  * https://www.openssl.org/source/license.html
  */
 
+#ifdef OPENSSL_SYS_WIN_CORE
+#include <windows.h>
```

Comment:
<windows.h> is included in "e_os.h"... [Well, it's suggested that we stop doing that, but we are not there yet.]

Reply:
removing windows.h is failing the build with I assume warning treated as error:
C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared\TraceLoggingProvider.h(2311): warning C4013: 'EventRegister' undefined; assuming extern returning int

Reply:
You naturally have to move inclusion of TraceLoggerProvider.h below. If you mean it fails even then, because relevant interfaces are masked as result of e_os.h setting _WIN32_WINNT, then there is a commentary that explains how to deal with it.